src/sparseml/pytorch/sparsification/quantization/helpers.py

```diff
@@ -433,68 +708,92 @@ def remove_activation_qat_by_layer_name(module: Module, layer_class_names: List[
 
 
 def get_qat_qconfig(
-    symmetric_activations: bool = False,
-    symmetric_weights: bool = True,
+    symmetric_activations: Optional[bool] = None,
```

Comment:
why are we defaulting variables to None here instead of setting a default?

Reply:
See reply above.